src/utilities/references.py

```diff
@@ -23,12 +23,12 @@
     "GRCh38premrna_and_SARSCoV2": "GRCh38premrna_and_SARSCoV2",
     "arc-GRCh38": "refdata-cellranger-arc-GRCh38-2020-A-2.0.0",
     "arc-mm10": "refdata-cellranger-arc-mm10-2020-A-2.0.0",
+    "arc-mm10-mcherry" : "arc-mm10-mcherry",
     "gex-GRCh38": "refdata-gex-GRCh38-2020-A",
     "gex-mm10": "refdata-gex-mm10-2020-A",
     "gex-GRCh38-and-mm10": "refdata-gex-GRCh38-and-mm10-2020-A",
 }
 
-
```

Comment:
We should keep this new line.

Reply:
file in place and working! I've put the new line back 👍